products/Zephyr Squad/Authentication.md

```diff
@@ -0,0 +1,30 @@
+The API authentication is based on JSON Web Token (JWT), which is a key for accessing the API. Authenticating using JWT requires the following steps:
+
+* Generate a key
+* Make authenticated requests
+
+## Generate a Key
+
+For accessing the API, you must generate an access key in Jira. To generate an access token, click on your profile picture at the page bottom left, and choose the option “Zephyr Squad API keys". For more information, please check out the documentation.
```

Comment:
profile picture will not be on the portal page, so this should be explained that they need to navigate to Atlassian click on their profile.

Reply:
should 'check out the documentation' have a link?